packages/frontend/src/App.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import { NextUIProvider } from '@nextui-org/react';
 import { BrowserRouter } from 'react-router-dom';
+import { AuthProvider } from './hooks/useAuth';
 import Router from './pages/routes/Router';
 import { mainTheme } from './theme';
 
-interface AppProps {}
+interface AppProps { }
 
 const App: React.FC<AppProps> = () => {
   return (
     <NextUIProvider theme={mainTheme}>
-      <BrowserRouter>
-        <Router />
-      </BrowserRouter>
+      <AuthProvider>
+        <BrowserRouter>
+          <Router />
+        </BrowserRouter>
+      </AuthProvider>
     </NextUIProvider>
+
```

Comment:
Did the prettier plugin generate this blank line for some reason?

Reply:
I will delete it. The line is there due to solving a conflict that I had when rebasing